### DISCUSSION TITLE
Idea: Common Tags and Resources boards v2.0

### DISCUSSION THREAD

**Discussion by ssimeonoff** [bafolts/terraforming-mars#1017] [Ideas]
### Reason
Before taking an action, often you scroll up to check an opponent resources or tags count. This causes a lot of scrolling and keeping the other players tableau open.

### Idea
Display the opponents tags and resources close to the player's resources and tags. The assumption is that if the player can see an overview of the opponents' resources and tags, and with the actions log visible, the opponents' tableau will remain closed, which will save space and avoid scrolling.

### v2.0
The idea is good but my previous suggestion (#1009)  was not.
Instead of adding and changing a lot UI, simply add toggle buttons that shows/hide the played cards within the opponent's tableau. The simple feature will solve the need the player and opponents resources/tags to be in view. And the "X" button already toggles on/off the tableau (if the player wants a clearer view).

Current UI:
![1](https://user-images.githubusercontent.com/6917565/87856174-96d50980-c91d-11ea-8af5-4799a5199d95.png)

Suggested UI:
![2](https://user-images.githubusercontent.com/6917565/87856408-42cb2480-c91f-11ea-8f23-d2ea1b788f06.png)

### Notes
- when button "show played cards" is pressed, that opponent's played cards are displayed and the button changes label to "hide played cards"

**Comment by meussiebeaucoup** [comment]
In my custom build for our games I've added a 'Tag Overview' section:

![image](https://user-images.githubusercontent.com/68510105/87879490-68bdfb00-c9eb-11ea-90bc-46b0c278088e.png)

We find that it works pretty well as it saves a lost of time of scrolling through all the players their tableau's. Of course it could use some better styling, but for a first draft I was pretty happy with how it turned out.

I can tidy it up and turn it into a pull request if you think this functionality is desired

**Comment by alrusdi** [comment]
@meussiebeaucoup can you push your version to guthub? We will check your implementation and take ideas from here.

@ssimeonoff main idea here is to have a table to compare values faster. What if we introduce a button like "Others" to show a comparison table with given parameter (tags, production+resources, influence, awards and milestones scoring)?

**Comment by ssimeonoff** [comment]
A huge table with a lot of data will have the problem as the first idea: too much UI. 
If you are moving the tags in a common table then the resources can be displayed in a minimized way right under the player name:
![2](https://user-images.githubusercontent.com/6917565/88146833-a9409300-cbfc-11ea-8ff3-cc93872b072a.png)

Milestone and awards can show the players values as well: 
![4](https://user-images.githubusercontent.com/6917565/88149961-0fc7b000-cc01-11ea-9493-f1410975de35.png)

And influence can be displayed over the CURRENT EVENT as:
![5](https://user-images.githubusercontent.com/6917565/88150116-400f4e80-cc01-11ea-9373-1477defe6e4a.png)
(a difference icon to be used that looks as the one used for influence in the game)

**Comment by vsrisuknimit** [comment]
Having the current values for Milestone and Award as @ssimeonoff  suggested would be really cool!

**Comment by vincentneko** [comment]
> Having the current values for Milestone and Award as @ssimeonoff suggested would be really cool!

This one is well underway !

**Comment by vincentneko** [comment]
> > Having the current values for Milestone and Award as @ssimeonoff suggested would be really cool!
> 
> This one is well underway !

This one is done !

**Comment by Elonka** [comment]
As another idea of how to implement this, how about putting reduced versions of the player trays in the area to the right of the map?

Thiw would be the same as what is already at the top of each player's tableau:
* Their trays (that we already see elsewhere), with their money, resources, production, etc.
* Their tag quantities
* Their score

It would be helpful to see all players' information "at a glance" this way in one part of the screen, even if the information were in reduced size.
